ts/private/ts_lib.bzl

```diff
@@ -82,6 +82,9 @@ https://docs.aspect.build/rulesets/aspect_rules_js/docs/js_library#deps for more
         executable = True,
         cfg = "exec",
     ),
+    "declaration_transpile": attr.bool(
```

Comment:
Note that this is different then the macro `declaration_transpiler`, this is just a flag indicating if the declarations are generated outside the rule so the rule doesn't have to do anything.